beam-core/Database/Beam/Query/Extra.hs

```diff
@@ -0,0 +1,99 @@
+module Database.Beam.Query.Extra
```

Comment:
I don't think we need a new module for these. They're generally useful and probably belong with the default versions.